pkg/start/start.go

```diff
@@ -9,11 +9,14 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"runtime"
 	"text/template"
 	"time"
 
 	"github.com/lima-vm/lima/pkg/driver"
 	"github.com/lima-vm/lima/pkg/driverutil"
+	"github.com/lima-vm/lima/pkg/qemu"
+	"github.com/lima-vm/lima/pkg/qemu/entitlementutil"
```

Comment:
(I don't want to let `pkg/start` depend on a specific driver, but I'm going to merge this right now anyway, as the Homebrew issue https://github.com/lima-vm/lima/issues/1742 is critical. We can refactor this later.)